gradle.properties

```diff
@@ -1,4 +1,4 @@
-base_version=0.12.0
+base_version=0.14.0
```

Comment:
where is 0.13?

Reply:
I couldn't work out why my change to kestrel wasn't taking effect in develop. Bumping this ver again made it work 😄